lisa/transformers/kernel_source_installer.py

```diff
@@ -171,17 +192,26 @@ def _modify_code(self, node: Node, code_path: PurePath) -> None:
             self._log.debug(f"modifying code by {modifier.type_name()}")
             modifier.modify()
 
-    def _build_code(self, node: Node, code_path: PurePath) -> None:
+    def _build_code(self, node: Node, code_path: PurePath, kconfig_file: str) -> None:
         self._log.info("building code...")
 
         uname = node.tools[Uname]
         kernel_information = uname.get_linux_information()
 
-        result = node.execute(
-            f"cp /boot/config-{kernel_information.kernel_version_raw} .config",
-            cwd=code_path,
-        )
-        result.assert_exit_code()
+        if kconfig_file:
+            err_msg = f"cannot find config path: {kconfig_file}"
+            assert node.shell.exists(node.get_pure_path(kconfig_file)), err_msg
+            result = node.execute(
+                f"cp {kconfig_file} .config",
```

Comment:
Use Cp tool.

Reply:
Previous implementation was using execute() way to run command so followed the same. This would require change in cp.copy() implementation to add **cwd** parameter, I will do it and use cp.copy().

Reply:
done